_includes/footer.html

```diff
@@ -0,0 +1,48 @@
+		<footer class="container" id="footer" role="contentinfo">
+			<div class="grid">
+				<div class="cell-4">
+					<h5>Get Involved</h5>
+
+					<ul class="unstyled">
+						<li>
+							<a href="//webchat.freenode.net/?channels=limetext">Chat on <code>#limetext</code></a>
+						</li>
+						<li>
+							<a href="//github.com/limetext/lime/issues">Current Issues</a>
+						</li>
+					</ul>
+				</div>
+
+				<div class="cell-4">
+					<h5>Quick Links</h5>
+
+					<ul class="unstyled">
+						<li>
+							<a href="//github.com/limetext/lime">Source Code</a>
```

Comment:
This link is technically incorrect now that the repos have been split up. Probably best to send people to `github.com/limetext` instead.